Makefile

```diff
@@ -50,30 +52,81 @@ v$(VERSION).tar.$(SRC_EXT):
+	mkdir -p $(DEB_BUILD)
+	tar -C $(DEB_BUILD) --strip-components=1 --atime-preserve \
+	  -xpf _topdir/SOURCES/$(NAME)-$(VERSION).tar.$(SRC_EXT)
+	ln _topdir/SOURCES/$(NAME)-$(VERSION).tar.$(SRC_EXT) \
+	  $(DEB_TOP)/$(NAME)-$(DEB_VERS).tar.$(SRC_EXT)
+
+$(DEB_TOP)/.patched: $(DEB_BUILD)/$(NAME)-$(VERSION) $(PATCHES)
+	# extract patches for Debian
+	mkdir -p ${DEB_BUILD}/debian/patches
+	mkdir -p $(DEB_TOP)/patches
+	for f in $(PATCHES); do \
+	  rm -f $(DEB_TOP)/patches/*; \
+	  git mailsplit -o$(DEB_TOP)/patches < $$f; \
+	  fn=$$(basename $$f); \
+	  for f1 in $(DEB_TOP)/patches/*; do \
+		f1n=$$(basename $$f1); \
+		echo "$${fn}_$${f1n}" >> $(DEB_BUILD)/debian/patches/series ; \
+		mv $$f1 $(DEB_BUILD)/debian/patches/$${fn}_$${f1n}; \
+	  done; \
+	done
+	touch $(DEB_TOP)/.patched
+
 # see https://stackoverflow.com/questions/2973445/ for why we subst
 # the "rpm" for "%" to effectively turn this into a multiple matching
 # target pattern rule
 $(subst rpm,%,$(RPMS)): $(SPEC) $(SOURCES)
 	rpmbuild -bb $(COMMON_RPM_ARGS) $(RPM_BUILD_OPTIONS) $(SPEC)
 
-$(subst deb,%,$(DEBS)): $(SPEC) $(SOURCES)
-	# Unpack tarball
-	export TAR_OPTIONS="--owner=0 --group=0 --numeric-owner"
-	mkdir -p ${DEB_BUILD}
-	tar -C $(DEB_BUILD) --strip-components=1 --atime-preserve \
-	  -xpf _topdir/SOURCES/$(NAME)-$(VERSION).tar.$(SRC_EXT)
-	# patch tarball
-	for f in $(PATCHES); do \
-	  patch -d $(DEB_BUILD) -p1 < $$f; \
-	  done
-	tar -C _topdir/BUILD \
-	  -czf _topdir/BUILD/$(NAME)-$(VERSION).tar.$(SRC_EXT) \
-	  $(NAME)-$(VERSION)
+$(subst deb,%,$(DEBS)): $(DEB_TOP)/.patched
 	# debmake --binaryspec preping
 	cd $(DEB_BUILD); debmake --binaryspec "$(NAME):lib,$(NAME)-dev:dev"
+	# Fix some autogenerated templates
+	#  README.Debian may need to be manually created as a patch to upstream
+	#  control file
+	#  If you supply a control file before running debmake, you also have to
+	#  supply most of the files that are now autogenerated.
+	cp debian_control $(DEB_BUILD)/debian/control
+	#  changelog as much as possible for now
+	sed -i 's/ Closes: #nnnn//' $(DEB_BUILD)/debian/changelog
+	sed -i '/<nnnn/d' $(DEB_BUILD)/debian/changelog
+	#   copyright
+	sed -i '/This is meant only as a template example/,$$d' \
+	  $(DEB_BUILD)/debian/copyright
+	sed -i 's#url://example.com#$(PROJECT_URL)#' \
+	  $(DEB_BUILD)/debian/copyright
+	echo "On Debian Systems, the full text of the GNU General Public" \
+	  >> $(DEB_BUILD)/debian/copyright
+	echo "Licenses version 2 and 3 can be found in the files" \
+	  >> $(DEB_BUILD)/debian/copyright
+	echo '`/usr/share/common-licenses/GPL-2'"' and " \
+	  >> $(DEB_BUILD)/debian/copyright
+	echo '`/usr/share/common-licenses/GPL-3'"' and " \
+	  >> $(DEB_BUILD)/debian/copyright
+	# installed files
+	sed -i 's:^#usr/:usr/:g' \
+	  $(DEB_BUILD)/debian/$(NAME).install
+	sed -i 's:^#usr/i:usr/i:g' \
+	  $(DEB_BUILD)/debian/$(NAME)-dev.install
+	sed -i 's:^#usr/lib/*/p:usr/lib/*/p:g' \
+	  $(DEB_BUILD)/debian/$(NAME)-dev.install
```

Comment:
Combine this with the above two per the previous comment so that there is one edit to both files?

Reply:
No longer applicable.